windows/flutter/generated_plugins.cmake

```diff
@@ -0,0 +1,17 @@
+#
```

Comment:
What's with all these windows related files?

We're not creating a windows application, it's polluting the codebase